src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/_client_factory.py

```diff
@@ -5,46 +5,42 @@
 
 
 # MANAGEMENT CLIENT FACTORIES
-def get_monitor_management_client(_):
+def cf_monitor(_):
     from azure.mgmt.monitor import MonitorManagementClient
     from azure.cli.core.commands.client_factory import get_mgmt_service_client
     return get_mgmt_service_client(MonitorManagementClient)
 
 
-def get_monitor_autoscale_settings_operation(kwargs):
-    return get_monitor_management_client(kwargs).autoscale_settings
-
-
-def get_monitor_diagnostic_settings_operation(kwargs):
-    return get_monitor_management_client(kwargs).service_diagnostic_settings
+def cf_alert_rules(kwargs):
+    return cf_monitor(kwargs).alert_rules
 
 
-def get_monitor_alert_rules_operation(kwargs):
-    return get_monitor_management_client(kwargs).alert_rules
+def get_monitor_autoscale_settings_operation(kwargs):
```

Comment:
As I go through the commands yes because that's how the network and VM packages look.